ERCS/erc-7790.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 7790
+description: Algorithmically Generated Token Standard
+author: LEVI_104 (@chen4903), birch (@birchwork)
+discussions-to: <URL> TODO
```

Comment:
Please create a topic in Eth Magicians, it just needs to contain a link to this PR.
https://ethereum-magicians.org/

Reply:
I am a new user of magician, and I need to wait a few days before I have the permission to create a new topic.